renku/cli/dataset.py

```diff
@@ -96,12 +95,24 @@ def dataset(ctx, client, datadir):
     """Handle datasets."""
     ctx.meta['renku.datasets.datadir'] = datadir
 
-    if ctx.invoked_subcommand is None:
-        table = tabulate(
-            client.datasets.values(),
-            headers=['short_id', 'name', 'authors_csv', 'created'],
-        )
-        click.echo(table)
+    if ctx.invoked_subcommand is not None:
+        return
+
+    datasets = client.datasets.values()
+    if not datasets:
+        click.echo('No dataset records found.')
```

Comment:
This should probably go to `sys.stderr` (`err=True`).

```suggestion
        click.secho('No dataset records found.', fg='red', err=True)
```